### PR TITLE
[Issue #20400]-Add unit test for YamlSQLParserRuleConfigurationSwapper

### LINKED ISSUE #20400: Add more unit test for YamlSQLParserRuleConfigurationSwapper
Hi community,

This issue is for https://github.com/apache/shardingsphere/issues/20305.

## Aim

Add more unit tests for `YamlSQLParserRuleConfigurationSwapper ` to test its public functions to improve unit test coverage.

Unit test coverage can be viewed at: https://app.codecov.io/gh/apache/shardingsphere/blobs/master/shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/main/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapper.java

## Basic Qualifications

- Java
- Maven
- Junit

### PR DESCRIPTION
**Pull request by abhinav-koppula**
Fixes #20400 .

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import org.apache.shardingsphere.parser.yaml.config.YamlSQLParserCacheOptionRuleConfiguration;
+import org.apache.shardingsphere.parser.yaml.config.YamlSQLParserRuleConfiguration;
+import org.apache.shardingsphere.sql.parser.api.CacheOption;
+import org.junit.Test;
+
+public final class YamlSQLParserRuleConfigurationSwapperTest {
+    
+    @Test
+    public void assertSwapToYamlConfiguration() {
+        YamlSQLParserRuleConfiguration actualResult =
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java:36]: Can you rename actualResult to actual?

File: shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        assertThat(actualResult.getSqlStatementCache().getInitialCapacity(), is(4));
+        assertThat(actualResult.getSqlStatementCache().getMaximumSize(), is(7L));
+    }
+    
+    @Test
+    public void assertSwapToObjectWithDefaultConfig() {
+        YamlSQLParserRuleConfiguration configuration = new YamlSQLParserRuleConfiguration();
+        configuration.setSqlCommentParseEnabled(true);
+        SQLParserRuleConfiguration actualResult = new YamlSQLParserRuleConfigurationSwapper().swapToObject(configuration);
+        assertThat(actualResult.getParseTreeCache().getInitialCapacity(), is(128));
```
Comment (strongduanmu) [inline shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java:50]: Can you rename actualResult to actual?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java (hunk 2 above)
Comment (abhinav-koppula) [reply shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java]: done, https://github.com/apache/shardingsphere/pull/21290/commits/8daffcc452be4a3d21369cf19a5409845431f139

File: shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java (hunk 1 above)
Comment (abhinav-koppula) [reply shardingsphere-kernel/shardingsphere-parser/shardingsphere-parser-core/src/test/java/org/apache/shardingsphere/parser/yaml/swapper/YamlSQLParserRuleConfigurationSwapperTest.java]: done, https://github.com/apache/shardingsphere/pull/21290/commits/8daffcc452be4a3d21369cf19a5409845431f139

**Review by strongduanmu — APPROVED** [review]
Good job, merged.

**Pull request merged**